akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/javadsl/ActorTestKit.scala

```diff
@@ -35,6 +35,21 @@ object ActorTestKit {
   def create(): ActorTestKit =
     new ActorTestKit(scaladsl.ActorTestKit(TestKitUtils.testNameFromCallStack(classOf[ActorTestKit])))
 
+  /**
+   * Create a testkit from the provided actor system.
+   *
+   * It will use the provided [[akka.actor.typed.ActorSystem]] and create a ActorTestKit,
+   * e.g. threads will include the name of the provided actor system.
```

Comment:
this comment line isn't relevant for this create method but I see most are incorrect in this file so we can follow up with a fix for that